airbyte-workers/src/main/java/io/airbyte/workers/process/KubePodProcess.java

```diff
@@ -370,11 +371,13 @@ public KubePodProcess(String processRunnerHost,
     List<Container> containers = usesStdin ? List.of(main, remoteStdin, relayStdout, relayStderr, callHeartbeatServer)
         : List.of(main, relayStdout, relayStderr, callHeartbeatServer);
 
+    Map<String, String> podLabels = labels.addAll(AIRBYTE_POD_LABELS);
```

Comment:
@davinchia Map<String, String> doesnt have the method `addAll` can you validate this?

Reply:
oof, typed too fast.

I changed the structure so this no longer applies.